test/integration/dd-agent/serverspec/dd-agent_spec.rb

```diff
@@ -8,6 +8,11 @@
   it { should be_running }
 end
 
+# verifies that the agent is installed per-machine for windows
+describe windows_registry_key('HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Installer\UserData\S-1-5-18\Products\776BC1A520EDC164EBF87A0875775D9C'), :if => os[:family] == 'windows' do
```

Comment:
The registry key is different on my test machine so the test fails. Not sure how the key is generated so I'm not sure how we could test this reliably.